docs/lang/articles/advanced/layout.md

```diff
@@ -63,7 +38,7 @@ ti.root.place(x)
 x = ti.field(ti.f32, shape=())
 ```
 
-The following declares a 1D field of shape `3`:
+* Declare a 1-D field of length `3`:
```

Comment:
suggest to be consistent with code ;/
```suggestion
* Declare a 1-D field of shape `3`:
```

Reply:
Yep, we should keep the terms consistent!